README.md

```diff
@@ -4,14 +4,18 @@ A context menu entry for Dolphin to open VS Code at the current location.
 
 Works on the empty background or on a folder.
 
-
 ## Download
 
 <a href="https://store.kde.org/p/1413799/"><img alt="Download from the KDE Store" height="50px"
 src="https://img.shields.io/badge/KDE_Store-blue?style=for-the-badge&logo=kde"></a>
 <a href="https://github.com/Merrit/kde-dolphin-open-vscode/releases/latest"><img alt="Download from GitHub" height="50px"
 src="https://img.shields.io/badge/GitHub-blue?style=for-the-badge&logo=github"></a>
 
+## Install
+
```

Comment:
Should probably clarify this is only necessary if not downloading the automatic way through the store.

```suggestion
## Manual Install

If not downloading from the KDE Store, install manually:

```

Reply:
True, thanks for the attention to detail @Merrit  :-)